dev-docs/multipart-subscriptions-protocol.md

```diff
@@ -61,7 +59,9 @@ Some errors are part of an execution result. Results may include partial data, a
 }
 ```
 
-When the router encounters an error that is fatal and should lead to termination of the subscription, it will instead send a message with a top-level `errors` field, and then close the connection:
+## Fatal Errors
+
+When the router encounters an error that is fatal and should lead to termination of the subscription, `payload` is null:
```

Comment:
`payload: null` without `errors` seems semantically more similar to a heartbeat. Can we treat it as a heartbeat instead?